.github/workflows/ci.yml

```diff
@@ -67,9 +67,16 @@ jobs:
       matrix:
         platform: [win32, x64, arm64]
     env:
-      CONFIGURATION: Release
       PLATFORM: ${{ matrix.platform }}
     steps:
+      - name: Setup environment
+        run: |
+          BRANCH=$(echo $GITHUB_REF | cut -d'/' -f 3)
+          if [[ $BRANCH == 'develop' || $BRANCH == 'master' ]]; then
```

Comment:
I take this to mean that, if a PR uses 'develop' or 'master' for its branch name, CO would still run with the 'ReleaseLTCG' configuration, correct?

Reply:
Correct, I could ensure that the event is also not a PR to avoid this.

Reply:
Yeah, or if such information unavailable, perhaps a check on `$GITHUB_REPOSITORY_OWNER == 'OpenRCT2'` would work?

Reply:
`$GITHUB_REF` looks like `ref/pull/<pr_number>/merge` for PRs. `cut` takes the third part, so it's impossible for a PR to pass this check.

From the [docs](https://docs.github.com/en/actions/learn-github-actions/variables):
> The ref given is fully-formed, meaning that for branches the format is `refs/heads/<branch_name>`, for pull requests it is `refs/pull/<pr_number>/merge`, and for tags it is `refs/tags/<tag_name>`. For example, refs/heads/feature-branch-1.

Reply:
Yeah thats true, it would however still use LTCG on forks develop branches.

Reply:
I think its good that forks reflects what we do on the in this repo, LTCG is off for pulls even when its develop so I think this is fine as it is.